app/Server.hs

```diff
@@ -39,8 +39,16 @@ runServer = do
         -- Use line buffering to ensure logging messages are printed correctly
         hSetBuffering stdout LineBuffering
         hSetBuffering stderr LineBuffering
-        -- Set log level to INFO so requests are logged to stdout
-        updateGlobalLogger rootLoggerName $ setLevel INFO
+        -- Setup handlers for logger
+        filePath <- logFilePath
+        fileH <- fileHandler filePath INFO
+        stdoutH <- streamHandler stdout INFO
+        stderrH <- streamHandler stderr ERROR
+        -- Set log level to INFO so requests are logged
+        fileLogginEnabled <- logToFile
+        updateGlobalLogger rootLoggerName $ setLevel INFO . setHandlers
```

Comment:
It feels a bit odd to use `(.)` here. The meaning probably is more clear if you write this "imperatively" as two separate `updateGlobalLogger` calls, one for `setLevel` and one for `setHandlers`.